docs/api-types/grpc-route.md

```diff
@@ -27,7 +27,7 @@ This allows you to define and manage the routing of gRPC traffic within your Kub
 
 ### Annotations
 
-- `application-networking.k8s.aws/lattice-assigned-domain-name`
+- `application-networking.k8s.aws/lattice-assigned-domain-name`  
```

Comment:
nit: trailing space

Reply:
It is actually intended to create a line break. Seems like backslash doesn't work in mkdocs so this is an alternative to it